### PR TITLE
Fixes a small oversight in emote code

### PR DESCRIPTION
**Pull request by Niknakflak**
I was blackmailed and held at gunpoint to make this.  Be gentle with my soul.

### CODE REVIEW

File: code/modules/mob/living/emote.dm [hunk 1]
```diff
@@ -140,6 +140,7 @@
 /datum/emote/living/flip
 	key = "flip"
 	key_third_person = "flips"
+	message = "<B>[src]</B> does a flip
```
Comment (Iamgoofball) [inline code/modules/mob/living/emote.dm]: forgot the "

Comment (octareenroon91) [inline code/modules/mob/living/emote.dm]: Also I think only `"does a flip"` is needed

File: code/modules/mob/living/emote.dm [hunk 2]
```diff
@@ -140,6 +140,7 @@
 /datum/emote/living/flip
 	key = "flip"
 	key_third_person = "flips"
+	message = "<B>[src]</B> does a flip"
```
Comment (PKPenguin321) [inline code/modules/mob/living/emote.dm]: .

Comment (Niknakflak) [inline code/modules/mob/living/emote.dm]: descriptive comment thanks

Comment (PKPenguin321) [inline code/modules/mob/living/emote.dm]: put a PERIOD you CORGI

Comment (optimumtact) [inline code/modules/mob/living/emote.dm]: it's <b> you plebe

Comment (lzimann) [inline code/modules/mob/living/emote.dm]: Remove the <b>src... aswell

**Comment by lzimann** [comment]
I don't agree with flip having a text. Spamming flip right now is fun, and adding a text means you won't really be able to spam it without annoying everyone(and getting bwoinked for spamming the chat).

**Comment by evsey9** [comment]
no

**Comment by BeeSting12** [comment]
People spamming flip is gonna get annoying pretty damn fast. I'm good tbh

**Comment by lzimann** [comment]
Honestly, I'll go ahead and close this, I guess another maintainer can override this if they really want flip to have a text, but a big no from me.

**Comment by octareenroon91** [comment]
thank fuck

**Comment by Cyberboss** [comment]
Seriously though, rip fun by spamming this emote if this was merged

**Comment by PKPenguin321** [comment]
Can we remove the text from *spin for this reason too least

**Comment by PKPenguin321** [comment]
Please*

**Comment by optimumtact** [comment]
Yes

**Comment by Cyberboss** [comment]
No

**Comment by Cyberboss** [comment]
I got fucking autocorrected: yes

**Pull request closed without merging**